assets/sass/components/setup/_googlesitekit-setup-module.scss

```diff
@@ -66,12 +66,13 @@
 		margin-top: 0;
 	}
 
-	.googlesitekit-setup-module__inputs + .googlesitekit-settings-notice {
-		font-size: 0.875rem;
-		line-height: 1.42;
-		margin-top: 0;
-		padding: 14.5px;
-	}
+	// two week old rule by Asvin
+	// .googlesitekit-setup-module__inputs + .googlesitekit-settings-notice {
+	// 	font-size: 0.875rem;
+	// 	line-height: 1.42;
+	// 	margin-top: 0;
+	// 	padding: 14.5px;
+	// }
```

Comment:
@eugene-manuilov  This threw me off.  This rule was added by Asvin while I was doing this task :thinking: 

I've never used global CSS styles with React, I would control the size by a prop

Reply:
@asvinb can you help Daniel?

Reply:
@danielgent We are not using CSS in JS anywhere, if it's what you mean you would control the size by a prop. Can you let me know what the issue with the CSS because I'am not sure am following 😅 
Basically the styles were tweaked to follow the designs where the font size is smaller and overall padding less as per the designs here:
https://www.figma.com/file/ix0pcdTQi1Bf9QFNBlBByB/GA4-UA-onboarding?node-id=20269%3A8635

Reply:
@asvinb I see it as 14px everyone on the design (which is what this component has)

I'll leave in the `margin-top` part of the rule. Totally I can the reason to control margin from outside a component (although I prefer Chakra-UI's way of doing them with style props!

But I think digging into the internals of other components breaks the React abstraction of components and is a recipe for disaster with so many devs :yum: 

No drama though! Just some crossed wires with so many tickets in action :+1:

Reply:
The font-size should have been 14px everywhere and I didn't set it!  :joy: 

So nice spot there @asvinb  :muscle: